src/afl-cc.c

```diff
@@ -758,7 +758,7 @@ static void edit_params(u32 argc, char **argv, char **envp) {
     if (!strncmp(cur, "-fsanitize-coverage-", 20) && strstr(cur, "list="))
       have_instr_list = 1;
 
-    if (!strcmp(cur, "-fsanitize=address") || !strcmp(cur, "-fsanitize=memory"))
+    if (!(strcmp(cur, "-fsanitize=address") && strcmp(cur, "-fsanitize=memory")))
       asan_set = 1;
```

Comment:
the original line is correct. if either one is set then USE_ASAN/USE_MSAN is ignored as only one of them can be set.

Reply:
well they are the same thing, but i had originally changed it because i had added a check for something else, then removed it. i will change it back though ;)